egs/wsj/s5/steps/segmentation/internal/merge_segment_targets_to_recording.py

```diff
@@ -178,7 +178,7 @@ def run(args):
                 cmd = ("copy-feats --binary=false {mat_fn} -"
                        "".format(mat_fn=targets[utt]))
                 p = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE,
-                                     stderr=subprocess.PIPE)
+                                     stderr=subprocess.PIPE, universal_newlines=True)
```

Comment:
In the docs I see this:
`The universal_newlines argument is equivalent to text and is provided for backwards compatibility.`
https://docs.python.org/3/library/subprocess.html?highlight=subprocess

Reply:
I think this should be `text`, not `universal_newlines`.  And I don't  see how the changes to combine_ali_dirs.sh relate to this.